backend/tests/api/test_chat.py

```diff
@@ -148,3 +149,32 @@ def test_retrieve_correct_session_id(
         response = client.get("/chat/test_session1", headers=headers)
         assert response.status_code == 200
         assert len(response.json()) == 10
+
+
+class TestMultiturnChat:
+    @pytest.mark.parametrize(
+        "session_id, history_exists",
+        [("test_session1", True), ("test_session2", True), ("test_session3", False)],
+    )
+    async def test_chat_history_summary(
+        self,
+        chat_message: ChatUserMessageBase,
+        chat_history: None,
+        session_id: str,
+        history_exists: bool,
+        asession: AsyncSession,
```

Comment:
@sidravi1 I think the `async` nature of this test is prompting this deprecation warning for me. 
```
/pytest_asyncio/plugin.py:208: PytestDeprecationWarning: The configuration option "asyncio_default_fixture_loop_scope" is unset.
The event loop scope for asynchronous fixtures will default to the fixture caching scope. Future versions of pytest-asyncio will default the loop scope for asynchronous fixtures to function scope. Set the default fixture loop scope explicitly in order to avoid unexpected behavior in the future. Valid fixture loop scopes are: "function", "class", "module", "package", "session"
```

Reply:
Yes! I added a fix to the next PR in pyproject.toml to handle this. Thanks.